remote/remote_state_s3.go

```diff
@@ -25,9 +25,11 @@ import (
 type ExtendedRemoteStateConfigS3 struct {
 	remoteStateConfigS3 RemoteStateConfigS3
 
-	S3BucketTags         []map[string]string `mapstructure:"s3_bucket_tags"`
-	DynamotableTags      []map[string]string `mapstructure:"dynamodb_table_tags"`
-	SkipBucketVersioning bool                `mapstructure:"skip_bucket_versioning"`
+	S3BucketTags						[]map[string]string `mapstructure:"s3_bucket_tags"`
+	DynamotableTags					[]map[string]string `mapstructure:"dynamodb_table_tags"`
+	SkipBucketVersioning		bool                `mapstructure:"skip_bucket_versioning"`
+	SkipBucketSSEncryption	bool                `mapstructure:"skip_bucket_ssencryption"`
+	SkipBucketAccessLogging bool                `mapstructure:"skip_bucket_accesslogging"`
```

Comment:
Is this just GitHub rendering or is the indentation not quite right here? Try running `make fmt` if you haven't already.

Reply:
Weird - I use "Atom" v1.34.0 on Win10Home - looks nicely there. Installed MS VSC (latest) - modified it there, looks nicely - but again it's not fine in GitHub.
Will try `make fmt`

Reply:
Fixed it remotely on GitHub already.
Thank you for the nice advice though - it'll be useful in the future.

Reply:
Nah - I'll use the formatting from the Go Tools. 
It doesn't sound right to use GitHub remotely in a browser to fix such formatting issues.

Reply:
The funny part - after reverting it looks fine on GitHub as well... @GitHubSaySorryPlease